pkg/register/validation.go

```diff
@@ -0,0 +1,68 @@
+package register
```

Comment:
Why here and not in the endpoint as it has been done for the others api ?

Reply:
A test we decided to do with @harture. If this new middleware is validated, it will be a very long job to move all necessary code for management interface. That's why we decided to do this only in register.

Reply:
Okay got it. My opinion :  I do not think we need to do a middleware for validation and think we can remove it and make register the same as the others.